README.md

```diff
@@ -22,6 +22,44 @@ ember addon my-addon -b @embroider/addon-blueprint --yarn
 
 ### Options
 
+For all these options, you'll see a warning printed from `ember-cli` about unsupported options.
+`ember-cli` doesn't have a way to detect if flags are used by a blueprint.
+
+### `--pnpm`
```

Comment:
minor, I assume all these new sections should be of 4th level (one more than "Options" itself, like `--addon-location`)
```suggestion
#### `--pnpm`
```